Code/Tools/AssetProcessor/native/unittests/AssetProcessorManagerUnitTests.h

```diff
@@ -23,6 +23,9 @@ namespace AssetProcessor
         Q_OBJECT
 
     protected:
+        AssetProcessorManagerUnitTests() = default;
+        ~AssetProcessorManagerUnitTests();
```

Comment:
I don' think we can do that since the incomplete type AssetProcessorManager_Test (we uses pre-declaration and unique_ptr) is defined in the implementation file. The destructor needs to see the complete type of it during the destruction. Otherwise it causes the build failure on Linux.